pkg/smart/scsicommands.go

```diff
@@ -30,8 +30,14 @@ const (
 
 // SCSI Command Descriptor Block types are the various type of scsi cdbs which are used
 // to specify the various parameters or data required to send a particular scsi command
+
+// CDB6 type
```

Comment:
Can this be elaborated?

Reply:
Yes. How elaborate? Something like this is enough?

"// CDB6 type represents 6 byte CDB"
...

Reply:
or "// CDB6 is a slice of 6 BYTE"
... ?

Reply:
> or "// CDB6 is a slice of 6 BYTE"
> ... ?

Ok. I'll do this.
Thanks!